src/toggl_api.cc

```diff
@@ -1428,3 +1428,11 @@ void track_edit_size(void *context,
     }
     app(context)->TrackEditSize(width, height);
 }
+
+void toggl_iam_click(void *context,
```

Comment:
looks like this method wasn't added to the header file, so it can't be called from the clients.